subset/ipaddr/Dockerfile.test_ipaddr

```diff
@@ -0,0 +1,8 @@
+FROM daqf/aardvark:latest
+
+RUN $AG update && $AG install python python-setuptools python-pip netcat
+
+COPY subset/ipaddr/dhcp_tests.py .
+COPY subset/ipaddr/test_dhcp .
+
+CMD ["./test_dhcp"]
```

Comment:
should generally have newlines at end of file

Reply:
this one's fixed, but there are others...

Reply:
Actually, I looked through all Dockerfiles, conf files and python files and they don't(except for 1 conf file) have new lines at the end of the file. I want to keep them consistent; should I remove this new line, or add new lines to all files?